CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Backtrace Cocoa Release Notes
 
+## Version 2.0.5-beta2
```

Comment:
we're adding more work for us in the future. Can we please continue adding more changes to the Version 2.0.5?

Reply:
sure thing